docs/dev_tools/build_profiles.md

```diff
@@ -0,0 +1,21 @@
+# Build Profiles
+mbed 5.0 supports three primary build profiles, *default*, *debug* and *small*. The Online 
+Compiler uses the *default* profile. When building from the command-line,
+you can select the desired profile by adding the ```--profile <profile>```
+command-line flag. You can specify custom user defined profiles by giving the path
```

Comment:
I think "command-line" here is redundant, since we're already in the context of command-line.